### ISSUE TITLE
rst2man not found

### ISSUE THREAD

**Issue by henry0312** [pigoz/homebrew-mplayer2#8]
Hello.

I tried to install HEAD of unstable branch, [91a18f3998323b6d9b0ef67b4c1865d40fabfaf8](http://git.mplayer2.org/mplayer2/commit/?h=unstable&id=91a18f3998323b6d9b0ef67b4c1865d40fabfaf8)
But install failed.
https://gist.github.com/4012616

I checked where rst2man installed on terminal
`$ which rst2man` --> /usr/local/bin/rst2man

Why was the compile failed?

**Comment by henry0312** [comment]
Maybe is [c2eae54bfbdd31630a04b4e8ab6df6c3b6eb475c](http://git.mplayer2.org/mplayer2/commit/Makefile?h=unstable&id=c2eae54bfbdd31630a04b4e8ab6df6c3b6eb475c) wrong?

**Comment by pigoz** [comment]
that because of homebrew superenv. `env :std` in the formula should fix this.

**Comment by henry0312** [comment]
hm..
`brew install --env=std --HEAD pigoz/mplayer2/mplayer2 --with-libav`
I succeeded in building :D

**Comment by pigoz** [comment]
I added a few commits which should make the process more straightforward:

1 - I forcefully require the std env in the formula (commit: dddb3c7b). I hope something nicer will be available in homebrew in some time.
2 - I added a custom dependency on docutils. This will make homebrew complain of the missing dependency at the start of the build process and give a nicer help message to the user (commit: 28ec8f7c).

So.. I'm closing the issue. Let me know if you have any additional problems!
